pwem/objects/data.py

```diff
@@ -251,7 +253,11 @@ def standardize(self):
             self._defocusAngle.sum(180.)
         # At this point defocusU is always greater than defocusV
         # following the EMX standard
-        self._defocusRatio.set(self.getDefocusU() / self.getDefocusV())
+
+        if self.getDefocusV() != 0:
```

Comment:
It is not a good idea to use equality (or inequality) comparison on floating-point values. Maybe it should be tested with a given very small error (i.e close to zero)

Reply:
Done. I set the minimum of DefocusV as 0.1, do you think it is restrictive enough?